universal-login-react/test/core/services/WalletStorageService.ts

```diff
@@ -0,0 +1,70 @@
+import {StorageService, WalletStorageService} from '../../../src';
+import {ApplicationWallet, CounterfactualWallet} from '@universal-login/commons';
+import sinon from 'sinon';
+import {expect} from 'chai';
+
+describe('WalletStorageService', () => {
+  const counterfactualWallet: CounterfactualWallet = {
+    contractAddress: '0xFOO',
```

Comment:
A good practice is to use real parsable addresses.

Reply:
👍